pytorch_lightning/callbacks/finetuning.py

```diff
@@ -32,7 +32,7 @@
 log = logging.getLogger(__name__)
 
 
-def multiplicative(epoch):
```

Comment:
I'd keep the name so anybody reading the code knows what it is, can also add typing

Reply:
reverted